Branch-TestBed/Branch-TestBed/Branch-TestBed-Info.plist

```diff
@@ -45,6 +45,8 @@
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
 	</array>
+	<key>AppIdentifierPrefix</key>
+	<string>$(AppIdentifierPrefix</string>
```

Comment:
should this be `$(AppIdentifierPrefix)`? seems weird to have just an open parens